### PR TITLE
feat: Add landing page with carousel

### PR DESCRIPTION
**Pull request by DavidLiang01**
Created a landing page 
CU-mgznmy

### CODE REVIEW

File: client/src/components/CarouselPage.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  background: '#EDF2F7',
+  height: '75vh',
+  left: '50%',
+};
+
+const CarouselPage = (props) => {
+  return (
+    <div style={carouselStyle}>
+      <div>
+        <Text style={textStyle}>{props.title}</Text>>
```
Comment (gordonpn) [inline client/src/components/CarouselPage.jsx]:
```suggestion
        <Text style={textStyle}>{props.title}</Text>
```

File: client/src/constants.js [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+  {
+    title: 'Track your Shipments',
+    imgName: 'transport.jpg',
+  },
+  {
+    title: 'View your Orders',
+    imgName: 'vendor.jpg',
+  },
+  {
+    title: "Analyze and View your Company's Financial",
```
Comment (gordonpn) [inline client/src/constants.js]:
```suggestion
    title: "Analyze and View your Company's Finances",
```
What do you think of this?

File: client/src/constants.js [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+    imgName: 'accounting.jpg',
+  },
+  {
+    title: 'Forcast your Sales',
```
Comment (gordonpn) [inline client/src/constants.js]:
```suggestion
    title: 'Forecast your Sales',
```

File: client/src/constants.js [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+    imgName: 'planning.jpg',
+  },
+  {
+    title: "Place Customers' Purchases",
```
Comment (gordonpn) [inline client/src/constants.js]:
```suggestion
    title: "Manage Customers' Purchases",
```

File: client/src/components/CarouselPage.jsx (hunk 1 above)
Comment (DavidLiang01) [reply client/src/components/CarouselPage.jsx]: nice catch

**Comment by Mackbellemore** [comment]
Task linked: [CU-mgznmy UI - Add proper landing page](https://app.clickup.com/t/mgznmy)

**Review by sujvn — DISMISSED** [review]

**Review by sujvn — APPROVED** [review]

**Review by Mackbellemore — APPROVED** [review]

**Pull request merged**